code/modules/clothing/clothing.dm

```diff
@@ -73,6 +89,8 @@
 		LoadComponent(/datum/component/bloodysoles)
 	if(!icon_state)
 		item_flags |= ABSTRACT
+	if(has_shielding)
+		AddComponent(/datum/component/shielded, max_charges = max_charges, recharge_start_delay = recharge_delay, charge_increment_delay = recharge_rate, charge_recovery = recharge_amount, lose_multiple_charges = lose_multiple_charges, shield_icon = shield_icon)
```

Comment:
Why would you make value, then... the fuck. just write the numbers instead of values like
`AddComponent(/datum/component/shielded, 1, 2, 3, etc)`

Reply:
```suggestion
		AddComponent(/datum/component/shielded, 3, 20 SECONDS, 1 SECONDS, 1, FALSE, "shield-old")
```

Reply:
If i am not retarded this should work, and of course remove variables you created at top, they are useless

Reply:
wait i only realised now... why you initialize shielding at clothing? do that at clothing that had it before

Reply:
Because if I do that all shielded clothing have the exact same stats and they can't be changed for different subtypes.

Reply:
if it's going to be at a `/ctf` level (see above comment) i could take or leave the variables. i know in the future people want to expand ctf quite a bit more so it does make some sense even if we're not changing the values between all the types

Reply:
:-1: keep the variables because just putting a bunch of numbers and booleans is completely unreadable unless you dig into the source code of shielded

Reply:
it's not unreadable if he would've kept the named args and would've used a line per arg